resources/views/docs/info.md.blade.php

```diff
@@ -124,6 +124,16 @@
 v2      | current
 v1      | _legacy api provided by the old site, will be deprecated soon_
 
+### API Response Version
+
+Sometimes API response needs updating in non-backward compatible ways. In such cases, `x-api-version` header is used to determine which version of the response will be returned.
```

Comment:
forgot about this line 👀 
```suggestion
Sometimes, an API response need to be updated in non-backward compatible ways. In such cases, the `x-api-version` header is used to determine which version of the response will be returned.
```